torch/_inductor/config.py

```diff
@@ -31,7 +31,7 @@
 inplace_buffers = True
 
 # allow reusing buffers for more efficient memory use
-allow_buffer_reuse = True
+allow_buffer_reuse = False
```

Comment:
Intentional?

Reply:
Buffer reuse does need to be disabled for comms to work properly, but no, didn't mean to change the default.